src/components/nodes/BasicNode.vue

```diff
@@ -178,16 +178,27 @@
                     [this.$attrs.class]: true
                 }
             },
-            linkData() {
-                if(this.data.node.task) {
-                    return {link: VueFlowUtils.linkDatas(this.data.node.task, this.execution)}
+            cls() {
+                if (this.data.node.trigger) {
+                    return this.data.node.trigger.type;
                 }
-                return null
+
+                if (!this.data.node?.task) {
+                    return undefined;
+                }
+
+                return this.data.node.task.type;
             },
-            cls() {
-                return this.data.node?.task ? this.data.node.task.type : this.data.node?.trigger ? this.data.node.trigger.type : null
+            hoverTooltip() {
```

Comment:
No need to display the whole path, only the ID, way too much useless information however (the point of the tooltip is to display ID too long to fit in a node)
![image](https://github.com/kestra-io/ui-libs/assets/37600690/cfc7e6a2-63c8-4427-be26-088467a48cdf)

Reply:
I like the idea of easily understanding how nested we are with the hover but okay